File: LTHAM/086/086.cpp
#include <iostream>
#include <cmath>
using namespace std;

float Tong(float, int);

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	
	cout << "Tong la: " << Tong(x, n) << endl;
	return 0;
}

float Tong(float x, int n)
{
	float s = 0;
	float t = 1;
	int dau = -1;
	int i = 1;
	while (i <= n)
	{
		t = t * x * x;
		s = s + dau * t;
		dau = -dau;
		i++;
	}
	return s;
}

File: CTDK_Python/086/_086.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
t = 1
dau = -1
i = 2
while(i <= 2 * n):
    t = t * x * x
    s = s + dau * t
    i = i + 2
    dau = -dau

print("Tong la: ", s)




